setup.cfg

```diff
@@ -26,4 +26,4 @@ django_settings_module = config.settings.test
 ignore_errors = True
 
 [isort]
-known_third_party = allauth,celery,crispy_forms,dateparser,dateutil,django,django_filters,django_rest_passwordreset,djangoql,djqscsv,drf_extra_fields,drf_yasg,dry_rest_permissions,environ,fernet_fields,freezegun,hardcopy,location_field,multiselectfield,partial_index,phonenumber_field,phonenumbers,pytz,ratelimit,requests,rest_framework,rest_framework_nested,rest_framework_simplejwt,sentry_sdk,simple_history
+known_third_party = allauth,boto3,celery,crispy_forms,dateparser,dateutil,django,django_filters,django_rest_passwordreset,djangoql,djqscsv,drf_extra_fields,drf_yasg,dry_rest_permissions,environ,fernet_fields,freezegun,hardcopy,location_field,multiselectfield,partial_index,phonenumber_field,phonenumbers,pytz,pywebpush,ratelimit,requests,rest_framework,rest_framework_nested,rest_framework_simplejwt,sentry_sdk,simple_history
```

Comment:
Looks like this PR adds `boto3 ` and `pywebpush` but it's not clear why.

Reply:
we use both these libraries now